src/TanzuToolkit.sln

```diff
@@ -24,6 +24,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tanzu.Toolkit.CloudFoundryA
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tanzu.Toolkit.VisualStudioExtension.Tests", "VisualStudioExtension\test\Tanzu.Toolkit.VisualStudioExtension.Tests.csproj", "{443333BA-74FC-4150-87CB-AE174B89BDD5}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CFSsh", "CFSsh\CFSsh.csproj", "{3EB9CBAE-8032-4051-9A31-99B108DD019D}"
```

Comment:
Maybe not the right place to add this comment, but I wonder if we should name the new project CfSshWrapper to avoid implying it does more (like a re-implementation)?

Reply:
+1 to Tim's comment.  Note also the case change: `CF` to `Cf`.  Convention is to use camel case, even for abbreviations.

Reply:
"Wrapper" makes sense to me, but I thought we talked last week about only camel casing acronyms when they're 3 or more characters long, otherwise keeping them caps. That's the only reason I used `CF` here; happy to convert to `Cf` if I misunderstood that conversation last week (my muscle memory much prefers camel case anyway 😁)

Reply:
Resolved in [a65d952](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/pull/360/commits/a65d9520f4d84d864ec62aff70c81de6a6491a82)